### PR TITLE
Update Venetian.xml

### PR DESCRIPTION
**Pull request by conky77**

### CODE REVIEW

**Comment by donho** [comment]
@conky77 
Thank you for your update. 
Could you check the diff from v7.8.9 to the newest `english.xml` to make sure there's nothing's missing?
http://download.notepad-plus-plus.org/repository/MISC/diff/english.diff.htm

**Comment by conky77** [comment]
> @conky77
> Thank you for your update.
> Could you check the diff from v7.8.9 to the newest `english.xml` to make sure there's nothing's missing?
> http://download.notepad-plus-plus.org/repository/MISC/diff/english.diff.htm

@donho 
Done, and at the same time, I fixed some mistakes and improved some translation. I don't understand why the system takes such a long time to check the file.

File: PowerEditor/installer/nativeLang/venetian.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <!--
-    Venetian translation for Notepad++ 7.8.8
-    Last modified Sat, July 4th, 2020.
+    Venetian translation for Notepad++ 7.8.10
```
Comment (donho) [inline PowerEditor/installer/nativeLang/venetian.xml:3]: Next version will be 7.9

File: PowerEditor/installer/nativeLang/venetian.xml [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+    Last modified Thu, September 8th, 2020.
     For updates, see https://github.com/notepad-plus-plus/notepad-plus-plus/tree/master/PowerEditor/installer/nativeLang
 -->
 <NotepadPlus>
-    <Native-Langue name="Vèneto" filename="venetian.xml" version="7.8.8">
+    <Native-Langue name="Vèneto" filename="venetian.xml" version="7.8.10">
```
Comment (donho) [inline PowerEditor/installer/nativeLang/venetian.xml:8]: 7.9

File: PowerEditor/installer/nativeLang/venetian.xml [hunk 3]
```diff
@@ -975,25 +1048,25 @@ Te poi definire vari deƚimitatori de coƚona uxando spasi par spartire i divers
...
-                <Item id="2025" name="Otaƚe"/>
-                <Item id="2026" name="Exadesimaƚe"/>
-                <Item id="2027" name="Binario"/>
+                <Item id="2023" name="&amp;Testo da inserire"/>
+                <Item id="2033" name="&amp;Nùmaro da inserire"/>
+                <Item id="2030" name="Nùmaro &amp;inisiaƚe: "/>
+                <Item id="2031" name="Increment&amp;a de: "/>
+                <Item id="2035" name="&amp;Xeri inisiaƚi"/>
+                <Item id="2036" name="&amp;Ripeti: "/>
+                <Item id="2032" name="&amp;Formato"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/venetian.xml:1057]: Sorry I just fixed an issue regarding it: https://github.com/notepad-plus-plus/notepad-plus-plus/commit/55d671719c8f00f02ed32b15575bc2e6587a5e30
`&amp;` in front of 'F' should be removed.

**Review by donho — CHANGES_REQUESTED** [review]
Please remove `&amp;` in front of 'F'.

File: PowerEditor/installer/nativeLang/venetian.xml (hunk 3 above)
Comment (conky77) [reply PowerEditor/installer/nativeLang/venetian.xml]: done

**Review by donho — APPROVED** [review]

**Pull request closed without merging**